saleor/static/dashboard-next/page/views/PageUpdateForm.tsx

```diff
@@ -70,90 +77,126 @@ export class PageUpdateForm extends React.Component<
             <TypedPageDeleteMutation mutation={pageDeleteMutation}>
               {(
                 deletePage,
-                { called, data, error, loading: deleteInProgress }
+                { called, data: deleteResult, error, loading: deleteInProgress }
               ) => {
                 if (called && !deleteInProgress) {
                   if (error) {
                     console.error(error);
                     return;
                   }
-                  if (data && data.pageDelete.errors.length) {
+                  if (
+                    deleteResult &&
```

Comment:
Do we need to check if the result exists if we know that the mutation was called, finished loading and did not yield errors?

Reply:
There is wisdom in those words 🤔